static/src/components/builder/store/modules/taskPresenter.js

```diff
@@ -0,0 +1,44 @@
+import * as types from '../types';
+import utils from '../../utils';
+
+const initialState = () => {
+  return {
+    allowSaveWork: false,
+    autoSaveSeconds: 0,
+    allowAssignToUser: false
+  };
+};
+
+export const state = {
+  ...initialState()
+};
+
+export const getters = {
+    [types.GET_TASK_PRESENTER_PROPS] (state) {
+      return state;
+    }
+  };
+
+  export const mutations = {
+    [types.MUTATE_ALLOW_SAVE_WORK] (state, payload) {
+      state.allowSaveWork = payload;
+    },
+    [types.MUTATE_AUTO_SAVE_SECONDS] (state, payload) {
+      state.autoSaveSeconds = payload;
+    },
+    [types.MUTATE_ALLOW_ASSIGN_TO_USER] (state, payload) {
+      state.allowAssignToUser = payload;
+    },
+    [types.MUTATE_CLEAR_TASK_TIMER_FORM] (state, payload) {
```

Comment:
This definition may be a [duplicate](https://github.com/bloomberg/pybossa-default-theme/blob/main/static/src/components/builder/store/modules/taskTimer.js#L22) from the task-timer component. Could we verify? Thanks!